nnvm/python/nnvm/compiler/build_module.py

```diff
@@ -32,6 +32,8 @@ class BuildConfig(object):
     defaults = {
         "opt_level": 2,
         "add_pass": None,
+        "extra_lib_op": None,
```

Comment:
One reason to use extra here is that instead of returning a single lib, now there is an extra lib which compiles from the split operator. Both of the names make sense in some aspects. Since this is not a general complete solution, I name it as extra for now.